Functional.Enumerables.Extensions/ReplayableEnumerable.cs

```diff
@@ -1,9 +1,7 @@
-﻿using System;
-using System.Collections;
+﻿using System.Collections;
 using System.Collections.Generic;
-using System.Text;
 
-namespace Functional
+namespace Functional.Enumerables.Extensions
```

Comment:
Added namespace clarification to avoid class name collisions with the `ReplayableEnumerable` class I needed to copy into the other project.  Figured it would be safe since it is an `internal` class.

Reply:
Oh, hmm. Good call. It would probably make sense to eventually move all internal classes into namespaces for their respective projects.